mdata/cache/cache.go

```diff
@@ -0,0 +1,107 @@
+package cache
+
+import (
+	"flag"
+	"sync"
+
+	"github.com/raintank/metrictank/iter"
+	accnt "github.com/raintank/metrictank/mdata/cache/accnt"
+	"github.com/rakyll/globalconf"
+)
+
+var (
+	maxSize uint64
+)
+
+func init() {
+	flags := flag.NewFlagSet("chunk-cache", flag.ExitOnError)
+	// (1024 ^ 3) * 4 = 4294967296 = 4G
+	flags.Uint64Var(&maxSize, "max-size", 4294967296, "Maximum size of chunk cache in bytes")
+	globalconf.Register("chunk-cache", flags)
+}
```

Comment:
actually now you're making me unsure. the reason why i said it works is because i saw that the default values gets set the way i expected it to. but now that i think about it, at the time this gets executed probably the `globalconf` module doesn't even know the path of the config file yet, because that gets set in `main()`. let me test by setting that value in the config file